package com.utp.proyectoFinal.exception;

public class TestAlreadyStartedException extends RuntimeException {
    public TestAlreadyStartedException(String message) {
        super(message);
    }
} 